client/src/components/AlumniMentorship.js

```diff
@@ -88,6 +166,93 @@ export default class AlumniMentorship extends Component {
     handleMenuClick = (e, { id }) => this.setState({ activeItem: id })
 
     render() {
+        let details = this.props.userDetails
+        let availabilities = details.availabilities;
+        let topics = details.topics;
+        let zoomLink = details.zoomLink;
+        availabilities = availabilities.map(timeSlot => {
+            timeSlot.text = `${timeSlot.day} ${timeToSlot[timeSlot.time]}`
+            return timeSlot
+        })
+
+        const timeAvailabilitiesUpdate = (
+            <>
+            <Button
+                primary
+                style={{'margin-right': '5px'}}
+                floated='right'
+                color="blue"
+                onClick={this.openTimePreferencesModal}
+            >
+                Update Time Availabilities
+            </Button>
+            <TimePreferencesModal
+                modalOpen={this.state.timePreferencesModalOpen}
+                timePreferences={availabilities || []}
+                closeModal={this.closeTimePreferencesModal}
+                id={details._id}
+            />
+            </>
+        )
+
+        const topicAvailabilitiesUpdate = (
+            <>
+            <Button
+                primary
+                style={{'margin-left': '2px'}}
+                floated='right'
+                color="blue"
+                onClick={this.openTopicPreferencesModal}
+            >
+                Update Topic Preferences
+            </Button>
+            <TopicPreferencesModal
+                modalOpen={this.state.topicPreferencesModalOpen}
+                topicPreferences={topics || []}
+                closeModal={this.closeTopicPreferencesModal}
+                id={details._id}
+            />
+            </>
+        )
+
+        const zoomLinkUpdate = (
+            <>
+             <Button
+                primary
+                style={{'margin-right': '5px'}}
+                floated='right'
+                color="blue"
+                onClick={this.openZoomUpdateModal}
+            >
+                Update Video Meeting Link
+            </Button>
+            <ZoomUpdateModal
+                modalOpen={this.state.zoomUpdateOpen}
+                zoomLink={zoomLink || ''}
+                closeModal={this.closeZoomUpdateModal}
+                id={details._id}
+            />
+            </>
+        )
+
+        let topicListItems = []
```

Comment:
nit: We could use [this list](https://react.semantic-ui.com/elements/list/#types-divided) to make this slightly more aesthetic